test/common/network/lc_trie_test.cc

```diff
@@ -20,7 +21,7 @@ class LcTrieTest : public testing::Test {
     std::vector<std::pair<std::string, std::vector<Address::CidrRange>>> output;
     for (size_t i = 0; i < cidr_range_strings.size(); i++) {
       std::pair<std::string, std::vector<Address::CidrRange>> ip_tags;
-      ip_tags.first = fmt::format("tag_{0}", i);
+      ip_tags.first = absl::Substitute("tag_{0}", i);
```

Comment:
```
test/common/network/lc_trie_test.cc:24:29: error: 'Substitute' is unavailable: There was 1 substitution argument given, but this format std::string is either missing its $0, or contains one of $1-$9
      ip_tags.first = absl::Substitute("tag_{0}", i);
```
 
Mac OS fails because of this. Should it be `tag_$0` ?

Reply:
Fixed.